test/e2e/integration/frontend-test/group.ts

```diff
@@ -460,9 +460,9 @@ describe('Group', () => {
     cy.get(appFrontend.group.subGroup).find(appFrontend.group.delete).click();
     cy.snapshot('group: delete-warning-popup');
 
-    cy.get(appFrontend.group.subGroup).find(appFrontend.group.popOverCancelButton).click();
+    cy.get(appFrontend.group.subGroup).find(appFrontend.group.popOverCancelButton).click({ force: true });
```

Comment:
I don't know why this failed. Cypress was complaining that the popover was clipped by overflow properties of one of its parents, but I could not see that this was the case. The popover was working just fine in the test and locally.